static/app/views/organizationGroupDetails/allEventsTable.tsx

```diff
@@ -87,9 +87,7 @@ const AllEventsTable = (props: Props) => {
       projectId={projectId}
       totalEventCount={totalEventCount}
       customColumns={['minidump']}
-      setError={() => {
-        (msg: string) => setError(msg);
-      }}
+      setError={(msg: string | undefined) => setError(msg || '')}
```

Comment:
```suggestion
      setError={(msg: string | undefined) => setError(msg ?? '')}
```
I think the nullish coalescing operator would be better here 👌

Reply:
Hmm, yeah I suppose logically that makes more sense, if the message isn't null or undefined (but still falsey) we would want to display it. Nice find!